docs/userguide/networking/menu.md

```diff
@@ -0,0 +1,22 @@
+<!--[metadata]>
++++
+title = "Network configuration"
+description = "Docker networking feature is introduced"
+keywords = ["network, networking, bridge, docker,  documentation"]
+type="menu"
+[menu.main]
+identifier="smn_networking"
+parent= "engine_guide"
+weight=7
++++
+<![end-metadata]-->
+
+# Docker networks feature overview
```

Comment:
note that `type=menu` means the user can never see this content - their browser will be redirected to the index.md page.